homeassistant/components/velux/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""
+
+Connects to VELUX KLF 200 interface
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.velux/
+"
+"""
+
+import logging
+import asyncio
+from homeassistant.helpers import discovery
+
+DOMAIN = "velux"
+SUPPORTED_DOMAINS = ['scene']
+_LOGGER = logging.getLogger(__name__)
+VELUX_MODULE = None
+
+REQUIREMENTS = ['pyvlx==0.1.1']
+
```

Comment:
[Configuration validation](https://home-assistant.io/developers/development_validation/) is missing.